src/test/java/com/epam/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.service;
+
+import com.epam.entity.User;
+import org.junit.jupiter.api.Test;
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceImplTest {
+    @Test
+    //simple unit test with mockito framework
+    void getUser() {
+        UserServiceImpl userService = new UserServiceImpl();
+        User user = mock(User.class);
```

Comment:
It is highly recommended to use annotations here, if possible. They increases readability so much